config/default/forms/contact/place-types.json

```diff
@@ -0,0 +1,5 @@
+{
+  "clinic": "Household",
+  "health_center": "Area",
```

Comment:
Can you check out Gareth's comments here @ https://github.com/medic/medic/issues/5884#issuecomment-529748113?

Reply:
Thanks @kennsippell , I've removed the XLS forms for all places, and kept the PLACE_TYPE templates.
Also keeping XML forms so that they can be uploaded without converting, as you had suggested.
https://github.com/medic/medic/pull/5927/commits/d957a0811fdbdf7b7e396b92640b94d81286250e